### ISSUE TITLE
Differences in CO2 between carbon simulation and CO2 simulation in GCClassic

### ISSUE THREAD

**Issue by msulprizio** [geoschem/geos-chem#2463]
### Your name

Melissa Sulprizio

### Your affiliation

Harvard

### What happened? What did you expect to happen?

In preparation for retiring the CH4, CO2, and tagCO simulations in favor of the carbon simulation (https://github.com/geoschem/geos-chem/issues/2387), we need to address differences that still remain between species in the carbon simulation and the standalone simulations. A 1-day simulation (Jan 1, 2019) comparison between the CO2 simulation and carbon simulation with CO2 reveals the following differences in surface CO2 concentrations.

<img width="693" alt="Screenshot 2024-09-13 at 11 55 19 AM" src="https://github.com/user-attachments/assets/9dc0e39e-bf04-4e81-b857-1c4e83e9ceba">

### What are the steps to reproduce the bug?

Generate the following run directories with the latest `dev/14.5.0` branch (tagged at 14.5.0-alpha.11):

1. gc_4x5_merra2_CO2
2. gc_4x5_merra2_carbon_CO2
3. gc_4x5_merra2_carbon

Run each simulation for one day and compare the output.

### What GEOS-Chem version were you using?

14.5.0-alpha.11

### What environment were you running GEOS-Chem on?

Local cluster

### What compiler and version were you using?

gcc 12.2.0

### Will you be addressing this bug yourself?

Yes

### In what configuration were you running GEOS-Chem?

GCClassic

### What simulation were you running?

Carbon

### As what resolution were you running GEOS-Chem?

4x5

### What meterology fields did you use?

MERRA-2

**Comment by yantosca** [comment]
Thanks @msulprizio.  It looks like some emissions are missing (CO2 SHIP) in the carbon single tracer simulation.

**Comment by msulprizio** [comment]
I found the `SHIP` logical was not defined in HEMCO_Config.rc for the carbon simulation, so ship emissions were not read in despite `CEDSv2_SHIP : true`. Fixing that reduced differences between the two simulations to:

<img width="711" alt="Screenshot 2024-09-13 at 12 12 56 PM" src="https://github.com/user-attachments/assets/ae6f8cf9-da6d-4e03-a441-a4d874037c10">

There were a few other inconsistencies in HEMCO_Config.rc:

1. The `DiagnFile` file was not defined for the CO2 simulation so no emissions diagnostics were saved out
2. An updated version of input file `Aviation_SurfCorr_SclFac.1x1.nc` was used in the carbon simulation (updated for COARDS compliance and use in GCHP). 

Fixing both of those did not impact the results of the CO2 simulation.

The remaining changes were able to be removed by turning off chemistry. Investigating further revealed that `CO2_COPROD` is true in the both simulations. I am exploring whether that source is double counted in the carbon simulation.

**Comment by msulprizio** [comment]
With the fixes now pushed to #2150, we are now able to reproduce species concentrations between the CO2 simulation and CO2-only carbon simulations.

<img width="660" alt="Screenshot 2024-10-16 at 4 52 42 PM" src="https://github.com/user-attachments/assets/2883b6c4-7233-4040-9b57-30374cb6f2ab">

**Comment by msulprizio** [comment]
I've changed the title of this issue to reflect that this issue tracks differences in GCClassic simulations (CO2 simulation vs CO2 in carbon simulations). GCHP differences will be tracked and fixed separately (see https://github.com/geoschem/GCHP/issues/437 and https://github.com/geoschem/GCHP/issues/440).

The fixes for CO2 in GCClassic carbon simulation from PR #2510 have now been merged into `dev/no-diff-to-benchmark` and will be included in 14.5.1.
